Integrations/Exabeam-new/Exabeam-new.py

```diff
@@ -0,0 +1,475 @@
+from typing import Dict
+
+import demistomock as demisto
+from CommonServerPython import *
+from CommonServerUserPython import *
+''' IMPORTS '''
+
+import json
+import requests
+from distutils.util import strtobool
+from datetime import datetime
+
+# Disable insecure warnings
+requests.packages.urllib3.disable_warnings()
+
+''' HELPERS '''
+
+
+class DemistoException(Exception):
+    pass
+
+
+class Client:
+    def __init__(self, exabeam_url, username, password, verify, proxies, headers):
+        self.server = exabeam_url.rstrip('/')
+        self.base_url = f'{self.server}/uba/api/'
+        self.username = username
+        self.password = password
+        self.verify = verify
+        self.proxies = proxies
+        self.headers = headers
+        self.session = requests.Session()
```

Comment:
why do we need a session?

Reply:
We must use session. Doesn't work otherwise